lib/pure/uri.nim

```diff
@@ -56,6 +85,10 @@ proc encodeUrl*(s: string, usePlus=true): string =
   ##
   ## As a special rule, when the value of ``usePlus`` is true,
```

Comment:
Github doesn't let me comment on line 81 (few lines above), where you have "All other characters are encoded as ``''%xx'`` where ``xx`` ..."

We should get rid of these single quotes inside of backticks, because what does `''%xx'` mean? These characters are encoded as `%xx` — without quotes. The same goes for `'+'` and `'%20'`: there are no quotes in the encoded url.

Reply:
Removed single quotes